src/Compilation/Config.zig

```diff
@@ -258,7 +260,7 @@ pub fn resolve(options: Options) ResolveError!Config {
             break :b false;
         }
 
-        if (options.lto == true) {
+        if (options.lto != null and options.lto != .none) {
```

Comment:
```suggestion
        if (options.lto) |x| if (x != .none) {
```

Nit: Slightly nicer way of writing these checks.